conjure-go-client/httpclient/internal/refreshingclient/dialer.go

```diff
@@ -20,56 +20,52 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/palantir/pkg/refreshable"
+	"github.com/palantir/pkg/refreshable/v2"
 	"github.com/palantir/witchcraft-go-logging/wlog/svclog/svc1log"
 	"golang.org/x/net/proxy"
 )
 
 type DialerParams struct {
 	DialTimeout   time.Duration
 	KeepAlive     time.Duration
-	SocksProxyURL *url.URL `refreshables:",exclude"`
+	SocksProxyURL *url.URL
 }
 
 // ContextDialer is the interface implemented by net.Dialer, proxy.Dialer, and others
 type ContextDialer interface {
 	DialContext(ctx context.Context, network, address string) (net.Conn, error)
 }
 
-func NewRefreshableDialer(ctx context.Context, p RefreshableDialerParams) ContextDialer {
-	return &RefreshableDialer{
-		Refreshable: p.MapDialerParams(func(p DialerParams) interface{} {
-			svc1log.FromContext(ctx).Debug("Reconstructing HTTP Dialer")
-			dialer := &net.Dialer{
-				Timeout:   p.DialTimeout,
-				KeepAlive: p.KeepAlive,
-			}
-			if p.SocksProxyURL == nil {
-				return dialer
-			}
-			proxyDialer, err := proxy.FromURL(p.SocksProxyURL, dialer)
-			if err != nil {
-				// should never happen; checked in the validating refreshable
-				svc1log.FromContext(ctx).Error("Failed to construct socks5 dialer. Please report this as a bug in conjure-go-runtime.", svc1log.Stacktrace(err))
-				return dialer
-			}
-			return proxyDialer
-		}),
-	}
-}
-
-// ConfigureDialer accepts a mapping function which will be applied to the params value as it is evaluated.
-// This can be used to layer/overwrite configuration before building the RefreshableDialer.
-func ConfigureDialer(r RefreshableDialerParams, mapFn func(p DialerParams) DialerParams) RefreshableDialerParams {
```

Comment:
Is this no longer needed, or is there an equivalent workflow now that `v2` is used?